src/fsharp/FSStrings.resx

```diff
@@ -861,6 +861,15 @@
   <data name="OverrideDoesntOverride4" xml:space="preserve">
     <value>The member '{0}' is specialized with 'unit' but 'unit' can't be used as return type of an abstract method parameterized on return type.</value>
   </data>
+  <data name="UnionCaseMissingArgument" xml:space="preserve">
+    <value>This constructor is missing arguments; 1 is expected.</value>
+  </data>
+  <data name="UnionCaseMissingArguments1" xml:space="preserve">
+    <value>This constructor is missing arguments; {0} are expected.</value>
+  </data>
+  <data name="UnionCaseMissingArguments2" xml:space="preserve">
+    <value> Patterns must unpack them, e.g. '{0}'</value>
```

Comment:
`, e.g. '{0}` should be: `, e.g., '{0}` (adding a comma)

Reply:
@cartermp done.